vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/base_rule.rb

```diff
@@ -0,0 +1,20 @@
+module SecureRooms
+
+  module AccessRules
+
+    class BaseRule
+
+      def self.call(user, card_reader, accounts, selected)
+        conditions_verdict = condition(user, card_reader, accounts, selected)
+        conditions_verdict || Verdict.new(:pass)
```

Comment:
I guess this handles it if the `condition` returns nil. I'm starting to feel like the extra method isn't really carrying its weight anymore, and we might as well have each of them call `call` directly rather than delegating out to `condition`.